src/gluonnlp/model/__init__.py

```diff
@@ -151,3 +152,117 @@ def get_model(name, **kwargs):
             'Model %s is not supported. Available options are\n\t%s'%(
                 name, '\n\t'.join(sorted(models.keys()))))
     return models[name](**kwargs)
+
+
+def get_tokenizer(model_name, dataset_name,
+                  vocab=None, root=os.path.join(get_home_dir(), 'models'),
```

Comment:
maybe also update this to:
root=os.path.join(get_home_dir(), 'data')

Reply:
You're right.